include/internal/refcount.h

```diff
@@ -70,15 +115,53 @@ typedef struct {
 
 static __inline__ int CRYPTO_UP_REF(CRYPTO_REF_COUNT *refcnt, int *ret)
 {
-    *ret = __atomic_fetch_add(&refcnt->val, 1, __ATOMIC_RELAXED) + 1;
+    *ret = __atomic_fetch_add(&refcnt->val, 1, __ATOMIC_ACQ_REL) + 1;
```

Comment:
Up refs should never need ACQ_REL

Reply:
thats an error, put that in the wrong place